### DISCUSSION TITLE
Does Ray display somehow the attachments of an email?

### DISCUSSION THREAD

**Discussion by chatisk** [spatie/laravel-ray#289] [Q&A] [answered]
Hello, I have a Laravel Notification where i use the method attachments() to provide sometimes a pdf with QR codes. When the notification is sent via mail, I get the email representation in ray, but I dont see any attachments with it. Does the package support displaying attachments?

Thanks in advance

**Comment by freekmurze** [comment, marked as answer]
Currently not, but we'll consider displaying attachment names in the future.
